rats-tls/README.md

```diff
@@ -91,6 +98,15 @@ Check msr 0x503, return value must be 0:
 sudo rdmsr 0x503s
 ```
 
+**Notice: special prerequisites for SEV remote attestation in software capability.**
+
+- software capability:
+  - [Kernel supports to get guest firmware handle in guest](https://github.com/haosanzi/attestation-evidence-broker/blob/master/hack/0001-KVM-X86-Introduce-KVM_HC_VM_HANDLE-hypercall.patch).
+  - The ccp driver supports to get attestation report with `ioctl` command, please refer to the following patches for more information.
+    - [Move the implementation of sev_get_attestation_report to ccp driver](https://github.com/haosanzi/attestation-evidence-broker/blob/master/hack/0002-KVM-SVM-move-the-implementation-of-sev_get_attestati.patch).
+    - [Implement SEV_GET_REPORT ioctl command in ccp driver](https://github.com/haosanzi/attestation-evidence-broker/blob/master/hack/0003-crypto-ccp-Implement-SEV_GET_REPORT-ioctl-command.patch)
```

Comment:
I suggest you create a README under the hack directory to talk about above words, and then refer to it here.